docs/iris/src/userguide/loading_iris_cubes.rst

```diff
@@ -234,12 +234,12 @@ then specific STASH codes can be filtered::
     :class:`iris.Constraint` reference documentation. 
 
 
-Constraining across the dateline
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Constraining a circular coordinate across its boundary
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Occasionally you may need to constrain your cube with a region that crosses either
-the dateline or the meridian. An example use-case of this is to extract the entire
-Pacific Ocean from a cube whose longitudes are bounded by the dateline.
+Occasionally you may need to constrain your cube with a region that crosses the
+boundary of a circular coordinate. An example use-case of this is to extract the
```

Comment:
@pelson I could add the words dateline and (anti?)meridian here? Something like

```...that crosses the boundary (this is often the dateline or the meridian) of a...```

Reply:
By all means mention dateline/meridian as much as you see fit, this is appropriate. It will be the primary case users will encounter.

Reply:
If you're happy to do so, then having those words helps google come up with helpful answers. Don't shy away from the antimeridian - it is a term that some people will use (others will prefer dateline). https://en.wiktionary.org/wiki/antimeridian

Reply:
I'm one who prefers "dateline" 😜

Reply:
Can you find a way to write it so that it doesn't matter what your preference is? (i.e. have them both) 😄

Reply:
You should use `date line` with a space (I've been lazy on that too in comments). And consider, the international date line is not exactly co-located with the antimeridian, it has some deviations, and therefore antimeridian may be more correct (although common usage of date line suggests its usage here would be perfectly acceptable, and I have no strong feeling on this).